int8 robot_computed_transform_id
int16[] matched_ids_from
int16[] matched_ids_to
bool[] transform_est_success
geometry_msgs/PoseWithCovariance[] separators
---
bool success
